prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -86,6 +92,7 @@ public override async Task<bool> ProcessRule(Enrollee enrollee)
 
             bool passed = true;
 
+            _logger.LogDebug($"Going to use {_collegeLicenceClient.GetType().Name} in PharmanetValidationRule");
```

Comment:
Testing code?

Reply:
Although this isn't exactly telemetry, from https://www.se-radio.net/2020/05/episode-409-joe-kutner-on-the-twelve-factor-app/  "And so in the book beyond the 12-factor app, Kevin makes this analogy to a space probe. Like your application is like this craft that you’re sending into space and it’s not coming back to earth. So if there’s a problem with it, when it’s in orbit, you can only rely on the instrumentation that you installed before it was launched. And so we need to think about our apps that run in the cloud the same way. If there’s a problem in production, you need to have the metrics, the telemetry, the tools that you need to debug that right there with it designed, you know, included with it upfront." ... thoughts?

Reply:
Neat article, and I agree logging and telemetry is important. But this isn't logging anything of use; this is logging the name of which college license client is used. It will log "CollegeLicenceClient" every time. That is not useful data.

Reply:
It can also log `DummyCollegeLicenceClient` depending on the situation (currently `OC_APP` value)

Reply:
Due to inability to change signature of `Startup.ConfigureServices`, I couldn't log just a single time.  But I moved the log statement to `SubmissionRulesService`